<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QWidget" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кредит</string>
  </property>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>381</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_procent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип платежей</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_payment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ежемесячный платеж</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_overpayment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Переплата по кредиту</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_payment_sum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Общая выплата</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sum_val">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_date_val">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_procent_val">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>50</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_sum_unit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date_unit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>месяцев</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_procent_unit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_payment_unit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_overpayment_unit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_payment_sum_unit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_payment_val">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_overpayment_val">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_payment_sum_val">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_type">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Annuitentnie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Differentials</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>350</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
